__tests__/app.test.js

```diff
@@ -49,4 +49,42 @@ describe("app", () => {
         });
     });
   });
+
+  describe("GET api/articles/:article_id", () => {
+    test("status 200: returns article by id", () => {
+      return request(app)
+        .get(`/api/articles/1`)
+        .expect(200)
+        .then(({ body }) => {
+          console.log(body, "body");
+          expect(body.article).toHaveProperty("article_id", expect.any(Number));
+          expect(body.article).toHaveProperty("title", expect.any(String));
+          expect(body.article).toHaveProperty("topic", expect.any(String));
+          expect(body.article).toHaveProperty("author", expect.any(String));
+          expect(body.article).toHaveProperty("body", expect.any(String));
+          expect(body.article).toHaveProperty("created_at", expect.any(String));
+          expect(body.article).toHaveProperty("votes", expect.any(Number));
+          expect(body.article).toHaveProperty(
+            "article_img_url",
+            expect.any(String)
+          );
+        });
+    });
```

Comment:
Great test. Don't forget to take out the console log! And one thing to note, instead of expecting any number for the article ID, you can directly assume it will be a the number that you're requesting in the url (in this case 1)